config/routes.rb

```diff
@@ -121,8 +121,15 @@
 
   # -------------------------------------------------------------- Static pages
   # jQuery Textile URLs
-  post '/preview' => 'home#textilize',  as: :preview
-  get '/markup-help' => 'home#markup_help', as: :markup
+  resource :textile, controller: :textile, only: [] do
+    collection do
```

Comment:
This `collection do` block doesn't actually do anything if we're still specifying `get` `post` inside it.